src/features/room/sagas/preselect.saga.tsx

```diff
@@ -134,7 +134,18 @@ export default function* preselect() {
                         throw new RoomNotFoundError(roomId)
                     }
 
-                    const { createdAt, createdBy, name = '' } = getRoomMetadata(stream)
+                    const {
+                        createdAt,
+                        createdBy,
+                        tokenAddress,
+                        name = '',
+                    } = getRoomMetadata(stream)
+
+                    if (tokenAddress) {
+                        // avoid pinning tokenGated rooms
```

Comment:
Anything blocking here, or just unsupported by the current flow? `preselect` includes a soft version of `pin`…